dotcom-rendering/src/components/Card/components/UL.tsx

```diff
@@ -16,6 +16,12 @@ const ulStyles = (direction: Direction) => css`
 		flex-direction: column;
 		width: 100%;
 	}
+
+	@supports not (row-gap: 12px) {
+		& > li {
+			margin-bottom: 12px;
+		}
+	}
```

Comment:
`@supports` itself is [not supported everywhere](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports#browser_compatibility), but the truly progressive alternative looked quite clunky:

```css
& > li {
	margin-bottom: 12px;
}

@supports (row-gap: 12px) {
	row-gap: 12px;
	& > li {
		margin-bottom: 0;
	}
}
```

Reply:
i would go with this version, personally. clunkiness > flakiness :)

plus a comment explaining the reason for it?